packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -0,0 +1,1352 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/Chart.yaml packages/rancher-monitoring/charts/Chart.yaml
+--- packages/rancher-monitoring/charts-original/Chart.yaml
++++ packages/rancher-monitoring/charts/Chart.yaml
+@@ -1,5 +1,9 @@
+ apiVersion: v1
+ appVersion: 0.38.1
++annotations:
++  catalog.rancher.io/certified: rancher
++  catalog.rancher.io/namespace: monitoring-system
++  catalog.rancher.io/release-name: rancher-monitoring
+ description: Provides easy monitoring definitions for Kubernetes services, and deployment
```

Comment:
We should update the description to add in what other charts that we're adding as part of our version of the chart.

Reply:
Updated to `A Rancher chart that modifies the upstream Prometheus Operator chart, which provides easy monitoring definitions for Kubernetes services and the deployment and management of Prometheus instances, and enables Prometheus Adapter on a default Prometheus instance.`